data/json/recipes/recipe_food.json

```diff
@@ -5848,7 +5852,9 @@
       [ "milk", 1 ],
```

Comment:
If coconut milk is always a valid replacement for milk we could consider a `using` material requirement set